Secure Score/Sensitive information types scanning should be enabled/Azure Policy - Audit/README.md

```diff
@@ -0,0 +1,28 @@
+# Azure Policy Sensitive information types scanning
+
```

Comment:
Please put all files under new directory called 'Private preview recommendations' in the main directory (and not under secure score)